tests/unit/mail_test.sh

```diff
@@ -494,6 +500,16 @@ function test_mail_send()
   expected='git send-email --to="mail@test.com" extra_args --other_arg -13 -v2'
   assert_equals_helper 'Testing no options option' "$LINENO" "$expected" "$output"
 
+  parse_mail_options "$FAKE_PATCH_FILE" '--to=mail@test.com'
```

Comment:
Also, I would like to see more real life cases:

1. More than one patch file without cover letter
2. More than one patch file with cover letter
3. Use of patch files with prefix `000` (as in `0003` or `0010`) and suffix `.patch` to relate to what `git format-patches` outputs